### PR TITLE
Swap Namespace, contracts on testnet

### PR DESCRIPTION
**Pull request by ggobugi27**
Yaml
- ns.yaml
- swap-callable.yaml
- tokens.yaml
- exchange.yaml

Pact
- ns.pact
- swap-callable.pact
- tokens.pact
- exchange.pact

### CODE REVIEW

File: pact/ns.pact [hunk 1]
```diff
@@ -1,5 +1,8 @@
+(define-keyset 'swap-ns-user)
+(define-keyset 'swap-ns-admin)
+(ns.write-registry 'swap (keyset-ref-guard 'swap-ns-admin) true)
```
Comment (slpopejoy) [inline pact/ns.pact:3]: This will break in `exchange.repl` tests no?

File: pact/exchange.pact [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 (module exchange GOVERNANCE
 
-  (defcap GOVERNANCE () (enforce false "autonomous"))
+  (defcap GOVERNANCE () (enforce (keyset-ref-guard 'swap-ns-admin)))
+  ; (defcap GOVERNANCE () (enforce false "autonomous"))
```
Comment (slpopejoy) [inline pact/exchange.pact:8]: Go ahead and delete these lines

File: pact/tokens.pact [hunk 3]
```diff
@@ -27,7 +27,8 @@
 
   (defconst ISSUER_KEY "I")
 
-  (defcap GOVERNANCE () (enforce false "autonomous"))
+  ; (defcap GOVERNANCE () (enforce false "autonomous"))
```
Comment (slpopejoy) [inline pact/tokens.pact:30]: delete

File: pact/yaml/testnet/swap-callable.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+codeFile: ../swap-callable.pact
```
Comment (slpopejoy) [inline pact/yaml/testnet/swap-callable.yaml:1]: This is fine, but I think I'd like to look at a pact upgrade that can concat files, so I might get that together today ...

**Review by slpopejoy — CHANGES_REQUESTED** [review]
Main issue is ensuring `exchange.repl` still works. We need to get some pact CI happening ...

File: pact/ns.pact (hunk 1 above)
Comment (ggobugi27) [reply pact/ns.pact:3]: Added ns.pact and loaded in the exchange.repl.

File: pact/exchange.pact (hunk 2 above)
Comment (ggobugi27) [reply pact/exchange.pact]: Deleted

File: pact/tokens.pact (hunk 3 above)
Comment (ggobugi27) [reply pact/tokens.pact]: Deleted

**Review by slpopejoy — APPROVED** [review]

**Pull request merged**